config.yaml

```diff
@@ -22,7 +22,9 @@ tft:
           - name: measure_cpu
           - name: measure_power
           - name: validate_offload
-        # Secondary network is required for tests 27, 28 and 29
+        # (Optional) Secondary network is required for tests 27, 28 and 29
```

Comment:
"(Optional) ... is required" is a bit of a contradiction.

Could you slightly reword? If the parameter is omitted in the YAML, it simply defaults to `f"{namespace}/ocp-secondary"`, so it's always there.

Maybe something like:
```
# Secondary network is required for tests POD_TO_POD_2ND_INTERFACE_SAME_NODE (27), POD_TO_POD_2ND_INTERFACE_DIFF_NODE (28), POD_TO_POD_MULTI_NETWORK_POLICY (29). If unspecified, defaults to "{namespace}/ocp-secondary".
```